libraries/app/application.cpp

```diff
@@ -351,6 +365,9 @@ namespace detail {
          }
          _chain_db->show_free_memory( true );
 
+         _next_rebroadcast = _chain_db->head_block_num() + 3 + rand() % 4;
+         _rebroadcast_con = _chain_db->applied_block.connect( [&]( const signed_block& b ){ rebroadcast_pending_tx(); } );
```

Comment:
Forgive my total ignorance of C++, but it is it necessary for the lambda to have access to all the variables in the enclosing scope?